manifests/g/GNU/gperf/3.1/GNU.gperf.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GNU.gperf
+PackageVersion: "3.1"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: gperf.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/carlescufi/winget-packages/raw/refs/heads/main/packages/gperf/3.1/gperf-3.1-win64_x64.zip
```

Comment:
This does not look like an official gperf distribution. Redistribution of packages is not allowed by this repo's policies

Reply:
Thanks for the review. It is not an official distribution. To my knowledge, there is no official gperf distribution for Windows, but you have allowed another version of gperf (an older one, 3.0.1) to be part of this repo:
https://github.com/microsoft/winget-pkgs/tree/master/manifests/g/GnuWin32/Gperf/3.0.1

The reasons I did not choose to update the GnuWin32 version to 3.1 instead are as follows:
- The Sourceforge GnuWin32 seems abandoned, has not been updated in 15+ years: https://gnuwin32.sourceforge.net/
- It uses an outdated installer mechanism
- It was not "official" to start with

Given all this, what are our options to get a more modern (i.e. 3.1 or later) version of GNU gperf as a winget package?

Reply:
cc @denelon

-----

My understanding is that `GnuWin32.gperf` is not considered a redistribution because 
1) The package is not claiming to be an official GNU package, it is under the identifier `GnuWin32`, which is its own separate company
2) The package URL and installer URL match the package identifier in being from GnuWin32

Reply:
The key is in having a set of official URLs showing the lineage from the original "author" / "owner" / "publisher" that lead to the distribution here in winget-pkgs. I did a bit of digging and it appears the source is coming from savannah project.

Reply:
The GitHub project this manifest points to doesn't include any licensing or attribution as far as I can tell to make sure individuals who go to take a look can see what source code was used to build the installer. Is that something you're planning on maintaining? As long as we have the right kinds of attributions and references to make sure we're not supporting any licensing violations, I'd lean in the direction of letting this in, but I'd want a few other folks to collaborate and make sure everything is done correctly 😊